packages/core/src/node/messaging/websocket-frontend-connection-service.ts

```diff
@@ -124,6 +125,16 @@ export class WebsocketFrontendConnectionService implements FrontendConnectionSer
     markForClose(channelId: string): void {
         this.channelsMarkedForClose.add(channelId);
     }
+
+    private frontendConnectionTimeout(): number {
+        const envValue = Number(process.env['FRONTEND_CONNECTION_TIMEOUT']);
+        if (!isNaN(envValue)) {
+            return envValue;
+        }
+
+        const configValue = BackendApplicationConfigProvider.get().frontendConnectionTimeout;
+        return configValue;
```

Comment:
```suggestion
        return BackendApplicationConfigProvider.get().frontendConnectionTimeout;
```

This is just a suggestion, ignore if you prefer the explicit variable, but I feel it doesn't add any additional clarity.